Sources/Kitura/OptionalCookieAttribute.swift

```diff
@@ -0,0 +1,87 @@
+import Foundation
+
+/// Describes an optional attribute of a cookie.
+public struct OptionalCookieAttribute {
```

Comment:
Having talked naming with @ianpartridge, we prefer the name `AdditionalCookieAttribute`, to avoid confusion with Swift's `Optional` type.  Please could you rename the type (and the source file)?